docs/sources/flow/reference/components/otelcol.exporter.loadbalancing.md

```diff
@@ -301,3 +395,565 @@ otelcol.exporter.loadbalancing "default" {
+  * Spans are received from instrumented applications via `otelcol.receiver.otlp`
+  * Spans are then exported via `otelcol.exporter.loadbalancing`.
+* A pool of "backing" Agents:
+  * The "backing" Agents run behind a headless service to enable the "load balancer" Agents to discover them.
+  * Spans are received from the "load balancer" Agents via `otelcol.receiver.otlp`
+  * Traces are then sampled via `otelcol.processor.tail_sampling`.
+  * The traces are exported via `otelcol.exporter.otlp` to a an OTLP-compatible database such as Tempo.
+
+{{< collapse title="Example Kubernetes configuration" >}}
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: grafana-cloud-monitoring
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: k6-trace-generator
+  namespace: grafana-cloud-monitoring
+spec:
+  minReadySeconds: 10
+  replicas: 1
+  revisionHistoryLimit: 1
+  selector:
+    matchLabels:
+      name: k6-trace-generator
+  template:
+    metadata:
+      labels:
+        name: k6-trace-generator
+    spec:
+      containers:
+      - env:
+        - name: ENDPOINT
+          value: agent-traces-lb.grafana-cloud-monitoring.svc.cluster.local:9411
+        image: ghcr.io/grafana/xk6-client-tracing:v0.0.2
+        imagePullPolicy: IfNotPresent
+        name: k6-trace-generator
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: agent-traces-lb
+  namespace: grafana-cloud-monitoring
+spec:
+  clusterIP: None
+  ports:
+  - name: agent-traces-otlp-grpc
+    port: 9411
+    protocol: TCP
+    targetPort: 9411
+  selector:
+    name: agent-traces-lb
+  type: ClusterIP
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: agent-traces-lb
+  namespace: grafana-cloud-monitoring
+spec:
+  minReadySeconds: 10
+  replicas: 1
+  revisionHistoryLimit: 1
+  selector:
+    matchLabels:
+      name: agent-traces-lb
+  template:
+    metadata:
+      labels:
+        name: agent-traces-lb
+    spec:
+      containers:
+      - args:
+        - run
+        - /etc/agent/agent_lb.river
+        command:
+        - /bin/grafana-agent
+        env:
+        - name: AGENT_MODE
+          value: flow
+        image: grafana/agent:v0.38.0
+        imagePullPolicy: IfNotPresent
+        name: agent-traces
+        ports:
+        - containerPort: 9411
+          name: otlp-grpc
+          protocol: TCP
+        - containerPort: 34621
+          name: agent-lb
+          protocol: TCP
+        volumeMounts:
+        - mountPath: /etc/agent
+          name: agent-traces
+      volumes:
+      - configMap:
+          name: agent-traces
+        name: agent-traces
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: agent-traces-sampling
+  namespace: grafana-cloud-monitoring
+spec:
+  clusterIP: None
+  ports:
+  - name: agent-lb
+    port: 34621
+    protocol: TCP
+    targetPort: agent-lb
+  selector:
+    name: agent-traces-sampling
+  type: ClusterIP
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: agent-traces-sampling
```

Comment:
I'd think this should match the text description, so it'd be `agent-traces-backing`.